tests/test_files/example_alert_data.yaml

```diff
@@ -0,0 +1,16 @@
058536939722682], [50.696816602381, -4.7827945997079695], [50.821953766351804, -4.647076874186137], [50.98475591553855, -4.6281211316914534], [51.165371409974256, -4.7154760497854795], [51.23074933217463, -4.695587177318256], [51.13748399104157, -4.569770135591332], [51.11561228889009, -4.349536259484776], [51.22777708641333, -4.237581413851652], [51.52315602649433, -4.281280671247355], [51.5661815493429, -4.331319436627314], [51.63722002031588, -4.647279274838982], [51.58675567579241, -5.034781391933181], [51.67624732471058, -5.301584675283319], [51.857405822071364, -5.3770118638630695], [52.05215701928441, -5.0781093115952025], [52.189534833196326, -4.479252799421595], [52.28129724874484, -4.2446724131164455], [52.39101199514596, -4.134312475369703], [52.64571910959783, -4.156280090391081], [52.8030125379852, -4.43657867149099], [52.73373527683138, -4.822902530825352], [52.927761154585966, -4.666562491312685], [53.3250019003421, -4.714885939734516], [53.42342331643293, -4.586885975410892], [53.45675230799499, -4.410215787662109], [53.33800749757923, -4.05068054471058], [53.365960580170906, -3.855255951886288], [53.32077210564533, -3.645315391765281], [53.38030802780183, -3.326527605009549], [53.58655662064125, -3.127665712038024], [53.90851674427833, -3.1306616988025278], [54.7296242877205, -3.9092366170518433], [54.58149719955063, -4.87876603437088], [54.61119595420949, -4.980185319772878], [54.82716025115902, -5.189796216669949], [54.672353717721734, -5.503138130818106], [54.47084275662029, -5.409812032887213], [54.304898288524974, -5.493032064508952], [54.00085787866954, -6.0624641709590765], [54.05664562710633, -6.1972216903862405], [54.01678406499502, -6.628795519864943], [54.23002998191055, -6.855873226656448], [54.19242565426902, -6.891642479581062], [54.21729620760012, -7.124037548551484], [54.09121885239279, -7.308294258459859], [54.10783178248695, -7.583960428694487], [54.18539905806525, -7.845223966277847], [54.44579647131303, -8.192958519674985]], [[57.75245668793269, -8.592820720527689], [57.798349418800925, -8.691519032735506], [57.86455485323485, -8.684557268975732], [57.91418787823128, -8.457317784625603], [57.83056845766009, -8.444037273308467], [57.75245668793269, -8.592820720527689]], [[59.112525647448926, -5.882765486506044], [59.181542047045404, -5.81291199632521], [59.10774665753379, -5.764033432697249], [59.065961116966065, -5.850962305785022], [59.112525647448926, -5.882765486506044]], [[60.12608338654564, -2.165529386790981], [60.34563034632679, -1.8219420500179084], [60.66077341109561, -1.470478899909446], [60.88793434492865, -0.9255934119305854], [60.84093307381703, -0.7139984990943649], [60.41704323871041, -0.6902305720104588], [60.27870185536764, -0.7717506307935432], [59.47160239287521, -1.5998545261997428], [59.508860947732025, -1.7073761021153309], [59.90779481600101, -1.6824886013593097], [60.09681539380499, -1.9172217831085037], [60.05915899413415, -2.0650519214591108], [60.12608338654564, -2.165529386790981]]]
```

Comment:
This isn't accurate anymore.

Here's the YAML we're planning to use for the Sizewell demo: https://github.com/alphagov/notifications-govuk-alerts/pull/65/files#diff-3d941c62395536004b33c30eb9f00d6b763d619dad61fb7522b8a3299e88a981

Reply:
useful, saw one additional key (`"starts"`) there